src/targets/docker.ts

```diff
@@ -55,10 +60,13 @@ export class DockerTarget extends BaseTarget {
         )
       );
     }
+
     return {
       password: process.env.DOCKER_PASSWORD,
       source: this.config.source,
       target: this.config.target,
+      sourceTemplate: this.config.sourceFormat || '{{source}}:{{revision}}',
+      targetTemplate: this.config.targetFormat || '{{target}}:{{release}}',
```

Comment:
_nit_: Should we make this `{{version}}`? I believe that's the terminology we mostly use across craft and repos. The "release" is what we create as a result.

Reply:
For example, we have this sort of interpolation in the `brew` target, where we support the variables `version`, `revision`, and `checksum`: 

https://github.com/getsentry/craft/blob/2c6c0bb523941abc30002f02f7a4d3fd70ced2fc/src/targets/brew.ts#L176-L178

Reply:
Good point, fix coming up.

Reply:
https://github.com/getsentry/craft/pull/126